RNReanimated.podspec

```diff
@@ -3,9 +3,7 @@ require_relative './scripts/reanimated_utils'
 
 reanimated_package_json = JSON.parse(File.read(File.join(__dir__, "package.json")))
 config = find_config()
-assert_no_multiple_instances(config)
 assert_latest_react_native_with_new_architecture(config, reanimated_package_json)
-
```

Comment:
Let's restore this newline here, it separates `assert_latest_react_native_with_new_architecture` call from envs.